src/main/scala/fi/aalto/cs/apluscourses/intellij/actions/ReplAction.scala

```diff
@@ -108,6 +110,18 @@ class ReplAction extends RunConsoleAction {
 
   }
 
+  def getDefaultModule(@NotNull project: Project): Option[Module] =
+    Option(ModuleManager.getInstance(project).findModuleByName(PluginSettings
+      .getInstance()
+      .getMainViewModel(project)
+      .courseViewModel
+      .get()
+      .getModel
+      .getAutoInstallComponentNames
+      //  we, hereby, commonly agree, that the first in the list auto install component (module)
+      //  is ultimately REPL's default module (as it's most likely to exist). sorry :pensive:
+      .head))
```

Comment:
The auto-install components list may also include non-module components, such as scala-sdk, but as long as we keep this special common agreement in mind, it's not an issue